env/wflow_noaacloud.env

```diff
@@ -0,0 +1,5 @@
+# Python environment for workflow on NOAA cloud
+
+setenv SRW_ENV regional_workflow
+
+conda activate regional_workflow
```

Comment:
This file follows a bit of a different approach to the one for hera, for example:
```
module load rocoto

module use -a /contrib/miniconda3/modulefiles
module load miniconda3
conda activate regional_workflow
```

Is miniconda managed with a module on NOAA Cloud?

Reply:
It is managed by modules there. I think it was also just in my path. I can update this to use the module explicitly, though.

Reply:
That would be great. It's nice to have all that managed via these files.